scripts/run.sh

```diff
@@ -9,7 +9,11 @@ if $RELEASE_MIRROR_ENABLED; then
     cp /etc/ssl/certs/github.com.crt /usr/local/share/ca-certificates/fake-github.com.crt
     update-ca-certificates
     echo "running nginx"
-    nginx -c /etc/nginx/nginx.conf
+    sudo nginx -c /etc/nginx/nginx.conf
 fi
 echo "starting istioctl commands"
 ./usr/local/app/scripts/create_istio_system.sh
+
+if $DEBUG; then
+    sleep 5m
```

Comment:
This should go above the Istio commands so they can be run manually. If they error it may never get here. 
you could also make debug an int and then sleep that amount of time